product_docs/docs/postgres_distributed_for_kubernetes/1/backup.mdx

```diff
@@ -111,34 +111,34 @@ spec:
 
 !!! Important
-    Currently, the retention policy will only be applied for the elected "Backup Node"
-    backups and WAL files. Given that each other PGD node also archives its own WALs
-    independently, it is your responsibility to manage the lifecycle of those WAL files,
-    for example by leveraging the object storage data retention policy.
-    Also, in case you have an object storage data retention policy set up on every PGD Node
+    Currently, the retention policy is applied only for the elected backup node
+    backups and WAL files. Given that every other PGD node also archives its own WALs
+    independently, it's your responsibility to manage the lifecycle of those WAL files,
+    for example by leveraging the object storage data-retention policy.
+    Also, in case you have an object storage data retention policy set up on every PGD node
     directory, make sure it's not overlapping or interfering with the retention policy managed
     by the operator.
 
 ## Compression algorithms
 
 Backups and WAL files are uncompressed by default. However, multiple compression algorithms are
-supported. For more information, refer to the [PG4K Compression algorithms](https://www.enterprisedb.com/docs/postgres_for_kubernetes/latest/backup_recovery/#compression-algorithms) documentation.
+supported. For more information, see the [PG4K compression algorithms](/postgres_for_kubernetes/latest/backup_recovery/#compression-algorithms) documentation.
 
 ## Tagging of backup objects
 
-It's possible to specify tags as key-value pairs for the backup objects, namely base backups, WAL files and history files.
-For more information, refer to the [PG4K document on Tagging of backup objects](https://www.enterprisedb.com/docs/postgres_for_kubernetes/latest/backup_recovery/#tagging-of-backup-objects).
+It's possible to specify tags as key-value pairs for the backup objects, namely base backups, WAL files, and history files.
+For more information, see the PG4K documentation about [tagging of backup objects](https://www.enterprisedb.com/docs/postgres_for_kubernetes/latest/backup_recovery/#tagging-of-backup-objects).
 
-## On-demand backups of a PGD Node
+## On-demand backups of a PGD node
 
-A PGD Node is represented as single-instance PG4K `Cluster` object.
+A PGD node is represented as single-instance PG4K `Cluster` object.
 As such, in case of need, it's possible to request an on-demand backup
-of a specific PGD Node by creating a PG4K `Backup` resource.
-In order to do that, you can directly refer to the [PG4K On-demand backups](https://www.enterprisedb.com/docs/postgres_for_kubernetes/latest/backup_recovery/#on-demand-backups) documentation.
+of a specific PGD node by creating a PG4K `Backup` resource.
+To do that, see the [PG4K on-demand backups](/postgres_for_kubernetes/latest/backup_recovery/#on-demand-backups) documentation.
```

Comment:
Another question about capitalization of resource name, if it's really a named thing.